src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-content: space-between;
+  width: 200px;
+  border: 1px solid #e5e5e5;
+  background-color: $color-white;
+
+  &__image {
+    margin: 32px 19px 0;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__info {
+    margin: 40px 16px 16px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__name {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $color-main-accent;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+    font-weight: 400;
```

Comment:
Is it default value? 
Use css-delegation. Apply this font-weight to .card or even .page